### PR TITLE
Create README-Polish.md

### PR DESCRIPTION
**Pull request by MichalLuck**
The Polish translation of the English REEADME.md file

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by danyilq** [comment]
You messed up flags and links for other languages at the end a little.
- [English] (../README.md)🇬🇧
- [中文 Chinese] (README-Chinese.md)🇨🇳
- [Polski] (README-Polish.md)🇵🇱
- [Українська] (README-Ukrainian.md)🇺🇦
Those should work.(just added spaces between name of link and link itself because it was formatting)

**Review by tomhamer — APPROVED** [review]

File: README-translated/README-Chinese.md [hunk 1]
```diff
@@ -259,4 +259,12 @@ Marqo是一个社区项目。我们的目标是让张量搜索能够被更多的
 - 加入我们的 [Slack社区](https://join.slack.com/t/marqo-community/shared_invite/zt-1d737l76e-u~b3Rvey2IN2nGM4wyr44w) 并和其他社区成员分享你的看法
 - Marqo 社区面对面 (正在筹备中!)
 
+**Translations:**
+
+- [English](../README.md)🇬🇧
+- [Français](README-French.md)
```
Comment (pandu-k) [inline README-translated/README-Chinese.md]: Can we add a French flag for consistency?

File: README-translated/README-Polish.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+### Forkers
+[![Forkers repo roster for @marqo-ai/marqo](https://reporoster.com/forks/marqo-ai/marqo)](https://github.com/marqo-ai/marqo/network/members)
+
+
+## Tłumaczenia
+
+Ten plik readme jest dostępny w następujących tłumaczeniach:
+
+- [English](../README.md)🇬🇧
+- [Français](README-French.md)
```
Comment (pandu-k) [inline README-translated/README-Polish.md]: French flag here too please

File: README.md [hunk 3]
```diff
@@ -267,4 +267,8 @@ Marqo is a community project with the goal of making tensor search accessible to
 
 This readme is available in the following translations:
 
+- [English](README.md)🇬🇧
+- [Français](README-translated/README-French.md)
```
Comment (pandu-k) [inline README.md]: French Flag here as well

**Review by pandu-k — CHANGES_REQUESTED** [review]
Can we please add French Flags to maintain consistency with the other translation links?

File: README-translated/README-Chinese.md (hunk 1 above)
Comment (MichalLuck) [reply README-translated/README-Chinese.md]: yeah ill go through it when i get on in the morning

File: README-translated/README-Polish.md (hunk 2 above)
Comment (MichalLuck) [reply README-translated/README-Polish.md]: kk!

File: README-translated/README-Chinese.md (hunk 1 above)
Comment (pandu-k) [reply README-translated/README-Chinese.md]: Thanks Michal!

**Review by pandu-k — APPROVED** [review]

**Pull request merged**